posthog/auth.py

```diff
@@ -231,9 +230,6 @@ def find_secret_api_token(
         if data and "secret_api_key" in data:
             return data["secret_api_key"], "body"
 
-        if "secret_api_key" in request.GET:
-            return request.GET["secret_api_key"], "query"
```

Comment:
I added the second arg of the tuple so that I could effectively count query param usage. The tuple is no longer needed so feel free to remove, otherwise I can address in a follow up PR.